scripts/portconfig

```diff
@@ -90,6 +91,9 @@ class portconfig(object):
         lag_tables = self.db.get_table(PORT_CHANNEL_TABLE_NAME)
         lag_mbr_tables = self.db.get_table(PORT_CHANNEL_MBR_TABLE_NAME)
         if port in port_tables:
+            port_transceiver_info = self.state_db.get_all(self.state_db.STATE_DB, "TRANSCEIVER_INFO|{}".format(port))
```

Comment:
should it just get "type" field here?

Reply:
no big difference here, and can avoid a case that TRANSCEIVER_INFO doesn't exist.